trappy/plotter/EventPlot.py

```diff
@@ -212,13 +207,28 @@ def _init_html(self):
                         "exports":  "EventPlot"
                     }
                 }
-            });
-            req(["require", "EventPlot"], function() {
-               EventPlot.generate('""" + self._fig_name + """', '""" + IPythonConf.add_web_base("") + """');
-            });
+            });"""
+
+        generate_call = """
+        req(["require", "EventPlot"], function() {
+            EventPlot.generate('""" + self._fig_name + """', '""" + IPythonConf.add_web_base("") + """', data);
+        });
```

Comment:
You don't have to assign it to a variable, just add the string straight away to `div_js` below